kuttl-test.yaml

```diff
@@ -19,7 +19,7 @@ kind: TestSuite
 reportFormat: JSON
 reportName: kuttl-test-ironic
 namespace: openstack
-timeout: 450
+timeout: 1000
```

Comment:
Do we really need such a high timeout? Note that if we then make a mistake in an assert such as a typo, we'll need to wait for the 1000 seconds to get the error

Reply:
yeah, sometimes locally we need this unfortunately.  It is not ideal for CI though I agree.  Maybe 750?

Reply:
If everything goes well it's not a problem, the timeout will only make the run longer when there is a problem. I think the worst case is when you want to test something quickly, but an assert is wrong. I would recommend to have it as low as possible and if needed increase in small jumps, but yeah, we can try 750 and see if it's enough